### ISSUE TITLE
Missing CONTRIBUTING.md

### ISSUE THREAD

**Issue by jshwlkr** [teachaccess/tutorial#5]
It might be useful to have a CONTRIBUTING.md file in the master branch to inform people how they might best contribute.

**Comment by nfreear** [comment]
Agreed. 👍  to a `CONTRIBUTING.md` file, and an explicit license ~ #6.

N.
